Tests/Core/Formatter/MultilineFormatterTests.swift

```diff
@@ -66,6 +66,35 @@ class MultilineFormatterTests: XCTestCase {
                          ]
                          """)
     }
+    
+    func testTupleString() {
+        let tupleElements: [(String?, String)] = [
+            (".0", "\"first\""), ("label","""
+            One(value: 1,
+                first: true)
+            """)
+        ]
```

Comment:
What about a this code style?

```suggestion
        let tupleElements: [(String?, String)] = [
            (
                ".0",
                "\"first\""
            ),
            (
                "label",
                """
                One(value: 1,
                    first: true)
                """
            )
        ]
```

Reply:
I adopt it in b10f02b, thx